lib/email-validation.rb

```diff
@@ -3,8 +3,12 @@
 require 'email-validation/config'
 
 module EmailValidation
-  def self.incorrect_email_message(email)
-    "The email address you have entered (#{email}) is incorrect."
+  def self.invalid_email_message
+    "Your email address is invalid. Please #{self.contact_link} if you need further assistance."
+  end
+
+  def self.could_not_verified_email_message
```

Comment:
I think this not very English :smile: Can we call it either `could_not_verify...` or `could_not_be_verified...`?